### PR TITLE
Sanitize OpenAPI spec by remove unwanted requestBody definitions from the delete endpoints

### PR DESCRIPTION
**Pull request by ayeshLK**
## Examples
N/A

## Checklist
- [ ] ~Linked to an issue~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: docs/spec/sanitations.bal [hunk 1]
```diff
@@ -112,6 +148,30 @@ function removeRequestBodyFromHttpGet(Specification spec) returns Specification
...
+            continue;
+        }
+
+        if REQUEST_BODY_NON_REQUIRED_OPS.indexOf(operationId) is () {
+            continue;
+        }
+
+        _ = delete.removeIfHasKey(REQUEST_BODY);
+        value.delete = delete;
+        paths['key] = value;
```
Comment (AzeemMuzammil) [inline docs/spec/sanitations.bal:170]: can we write this way?
```suggestion
Delete? delete = value.delete;
string? operationId = delete?.operationId;
if delete is Delete && operationId is string && REQUEST_BODY_NON_REQUIRED_OPS.indexOf(operationId) != () {
    _ = delete.removeIfHasKey(REQUEST_BODY);
    value.delete = delete;
    paths['key] = value;
}
```

Comment (ayeshLK) [reply docs/spec/sanitations.bal:170]: I would rather keep it as it is cos it is more clearer that way rather than a complex if-condition.

Comment (AzeemMuzammil) [reply docs/spec/sanitations.bal:170]: +1

**Review by AzeemMuzammil — APPROVED** [review]
LGTM

**Pull request merged**